src/ChooseArea.svelte

```diff
@@ -9,9 +9,9 @@
 
   let showAbout = false;
 
-  let inputValue;
+  let inputValue: string;
   let dataList;
```

Comment:
I couldn't easily figure out this type, will work on it later. Maybe there's a trick with VS Code to infer it?